SPECS/memtest86.spec

```
@@ -0,0 +1,35 @@
+# Debug package is empty and rpmlint rejects build
+%define _enable_debug_packages %{nil}
+%define debug_package %{nil}
+
+Summary:	A stand alone memory test for x86 architecture systems
+Name:		memtest86+
+Version:	6.01
+Release:	1%{?dist}
+License:	GPLv2
+Group:		System/Kernel and hardware
+Url:		http://www.memtest.org
+Source0:	https://github.com/memtest86plus/memtest86plus/archive/refs/tags/v%{version}.tar.gz
```

Better with a filename than matches the name of the directory it will unpack to:
```suggestion
Source0:	https://github.com/memtest86plus/%{name}/archive/v%{version}/%{name}-%{version}.tar.gz
```
(which means we need to rename the file too)

---

Wow, where did you get this link from, it is not the official one linked from [the release](https://github.com/memtest86plus/memtest86plus/releases/tag/v6.01)

---

It's a construct I probably got from Fedora packages a while ago when I was looking for a way to get URLs that are not lying about the actual filename and respect the %name-%version pattern.